README.md

```diff
@@ -19,19 +19,24 @@ You can <a href="https://files.digilent.com/Products/PYNQ/pynq_z1_image_2017_02_
 
 ## Quick Start
 
-See the <a href="http://pynq.readthedocs.io/en/latest/1_getting_started.html" target="_blank">Quickstart guide</a> for details on writing the image to an SD card, and getting started with the PYNQ-Z1 board.
+See the <a href="http://pynq.readthedocs.io/en/latest/getting_started.html" target="_blank">Quickstart guide</a> for details on writing the image to an SD card, and getting started with the PYNQ-Z1 board.
 
-## Modify Python
+## Python Source Code
 
-All Python code for the `pynq` package can be found in the `/python` folder. This folder can be found on the board after the board boots with the precompiled image. 
+All Python code for the `pynq` package can be found in the `/pynq` folder. This folder can be found on the board after the board boots with the precompiled image.
 
-You can clone this repository, edit the Python code and copy it directly to the corresponding folder on the board. (You may need to reboot the board to load the changes.)
+You can clone this repository, edit the Python source code and copy it directly to the corresponding folder on the board. (You may need to reboot the board to load the changes.)
+
+SDK software projects and Python-C source codes are also stored along with the Python source code. After installing the `pynq` package, the compiled target files will be saved automatically into the `pynq` package.
 
 ## Board Files and Overlays
 
-All board related files including Vivado and SDK software projects, bitstreams, and example notebooks, can be found in the `/Pynq-Z1` folder.
+All board related files including Vivado projects, bitstreams, and example notebooks, can be found in the `/boards` folder.
+
+You can rebuild the overlay by running *make* in the corresponding overlay folder (e.g. `/boards/Pynq-Z1/base`). This will generate a bitstream in the overlay folder. You can also find the project tcl file here. 
```

Comment:
Technically  - *make* is only for linux. In windows you need to source the appropriate tcl files. This is described in the RTD documentation. 

I'd leave off the *make* instruction or point to the docs